main/guides/getting-started/uicomponentlibrary.md

```diff
@@ -0,0 +1,112 @@
+# Welcome to the Agoric UI Component Library
```

Comment:
any particular reason to put this under `/getting-started/`?

Reply:
@dckc good catch - I want it to be a separate page on the same hierarchy as 'Getting Started' - not within it. @kbennett2000 might need your help to do that

Reply:
Sure, we can create a subfolder underneath `main/guides` for all of this